tests/app/celery/test_tasks.py

```diff
@@ -824,54 +778,22 @@ def test_should_process_email_job_with_sender_id(self, sample_email_template, mo
         """
         job = create_job(template=sample_email_template, sender_id=fake_uuid)
         mocker.patch("app.celery.tasks.s3.get_job_from_s3", return_value=email_csv)
-        mocker.patch("app.celery.tasks.save_email.apply_async")
+        mocker.patch("app.celery.tasks.save_emails.apply_async")
         mocker.patch("app.encryption.CryptoSigner.sign", return_value="something_encrypted")
         mocker.patch("app.celery.tasks.create_uuid", return_value="uuid")
 
         process_job(job.id)
 
-        tasks.save_email.apply_async.assert_called_once_with(
-            (str(job.service_id), "uuid", "something_encrypted"),
-            {"sender_id": fake_uuid},
-            queue="database-tasks",
+        tasks.save_emails.apply_async.assert_called_once_with(
+            (str(job.service_id), ["something_encrypted"], None), queue="-normal-database-tasks"
         )
 
-    @freeze_time("2016-01-01 11:09:00.061258")
-    def test_should_process_letter_job(self, sample_letter_job, mocker):
```

Comment:
I'm all for deleting letter code, by why this one?

Reply:
actually the code path doesnt make sense anymore...

Reply:
im just going to skip this test as this code path isn't around